tests/mocked/test_droplets.py

```diff
@@ -475,20 +509,31 @@ def test_droplets_list_backups(mock_client: Client, mock_client_url):
 
 
 @responses.activate
-def test_droplets_list_snapshots(mock_client: Client, mock_client_url):
+def test_list_snapshots(mock_client: Client, mock_client_url):
```

Comment:
Duplicated in https://github.com/digitalocean/pydo/pull/223, I would just remove this test.